Adjust docs about Prow service accounts (#673)
kyma-project/test-infra: docs/prow/authorization.md

| Service account name          | Usage                                                      | Required roles |
| :---------------------------- | :----------------------------------------------------------| :------------- |

> | **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket

Reviewer: plan --> plank

Author: done

> | **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket

Reviewer 2: ```suggestion
| **sa-gcs-plan**              | Used by Prow plan microservice. | `Storage Object Admin` (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
```

> | **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)

Reviewer 2: ```suggestion
| **sa-gke-kyma-integration**   | Runs integration tests on a GKE cluster. | `Compute Admin` (`roles/compute.admin`), `Kubernetes Engine Admin` (`roles/container.admin`), `Kubernetes Engine Cluster Admin` (`roles/container.clusterAdmin`), `DNS Administrator` (`roles/dns.admin`), `Service Account User` (`roles/iam.serviceAccountUser`), `Storage Admin` (`roles/storage.admin`)
```

> | **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin` (`roles/storage.objectAdmin`)

Reviewer: not only release artifacts but also development artifacts

Author: done

> | **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin` (`roles/storage.objectAdmin`)

Reviewer 2: ```suggestion
| **sa-kyma-artifacts**         | Saves release and development artifacts to the GCS bucket. | `Storage Object Admin` (`roles/storage.objectAdmin`)
```

Author: done

> | **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)` (`roles/compute.instanceAdmin`), `Compute OS Admin Login` (`roles/compute.osAdminLogin`), `Service Account User` (`roles/iam.serviceAccountUser`)

Reviewer 2: ```suggestion
| **sa-vm-kyma-integration**    | Runs integration tests on Minikube. | `Compute Instance Admin (beta)` (`roles/compute.instanceAdmin`), `Compute OS Admin Login` (`roles/compute.osAdminLogin`), `Service Account User` (`roles/iam.serviceAccountUser`)
```

> | **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Object Creator` (`roles/storage.objectCreator`) in the `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets

Reviewer 2: ```suggestion
| **sa-gcr-push-kyma-project**  | Publishes Docker images. | `Storage Object Creator` (`roles/storage.objectCreator`) in the `eu.artifacts.kyma-project.appspot.com` and `artifacts.kyma-project.appspot.com` buckets
```
